Packs/Exabeam/Integrations/Exabeam/Exabeam.yml

```diff
@@ -98,12 +109,47 @@ configuration:
   name: insecure
   type: 8
   required: false
+  section: Connect
+  advanced: true
 - display: Use system proxy settings
   name: proxy
   type: 8
   required: false
+  section: Connect
+  advanced: true
+- display: Fetch Type
+  defaultvalue: Exabeam Incident
+  name: fetch_type
+  type: 16
+  options:
+  - Exabeam Incident
+  - Exabeam Notable User
+  section: Collect
+- display: Max Users Per Fetch
```

Comment:
add additional info that its relevant only yo fetch users. to all the fetch user configurations